### ISSUE TITLE
is_training is not defined?

### ISSUE THREAD

**Issue by SeokLeeUS** [golbin/TensorFlow-Tutorials#45]
![image](https://user-images.githubusercontent.com/44729645/52737934-8471d600-2f9b-11e9-9424-1b3119163ee8.png)

batch normazation을 해보려고 하는데, is_training은 어떻게 설정을 하나요?
